app/celery/service_callback_tasks.py

```diff
@@ -58,26 +58,23 @@ def send_delivery_status_to_service(
     try:
         # calls the webhook / sqs callback to transmit message
         service_callback.send(payload=payload, logging_tags=logging_tags)
-    except RetryableException as e:
+    except RetryableException:
         try:
             current_app.logger.warning(
                 "Retrying: %s failed for %s, url %s.", self.name, logging_tags, service_callback.url
             )
-            current_app.logger.exception(e)
             raise AutoRetryException('Found RetryableException, autoretrying...')
         except self.MaxRetriesExceededError:
             current_app.logger.error(
                 "Retry: %s has retried the max num of times for %s, url %s.",
                 self.name, logging_tags, service_callback.url
             )
-            current_app.logger.exception(e)
-            raise e
-    except NonRetryableException as e:
-        current_app.logger.error(
+            raise
+    except NonRetryableException:
+        current_app.logger.critical(
             "Not retrying: %s failed for %s, url: %s. ", self.name, logging_tags, service_callback.url
         )
-        current_app.logger.exception(e)
-        raise e
+        raise
```

Comment:
Is this caught upstream?  What is the consequence?  On line 135, you deleted the "raise".

Reply:
I deleted all of them then hit issues and put them back, missed that one. Good catch! Changing the retry functionality was out of scope anyways, I just wanted to fix logging.